src/components/ThumbnailImage.js

```diff
@@ -75,6 +76,12 @@ class ThumbnailImage extends PureComponent {
         return {thumbnailWidth: thumbnailScreenWidth, thumbnailHeight: Math.max(40, thumbnailScreenHeight)};
     }
 
+    /**
+     * Update the state with the computed thumbnail sizes.
+     *
+     * @param {{ width: number, height: number }} Params - width and height of the original image.
+     * @returns {void}
```

Comment:
We don't usually mention returns annotation when there is nothing to return so can you remove this?

Reply:
Fixed.